### ISSUE TITLE
Add upload_set_id to default work when uploading files

### ISSUE THREAD

**Issue by awead** [projecthydra/sufia#1397]
This was introduced in 8aa663b0071af2e8d9c96c7ea30a47569c775337 but can be fixed once projecthydra-labs/curation_concerns#541 is closed.

**Comment by awead** [comment]
Superseded by #1398? I don't think you can redirect to the batch upload page without having set the upload_set_id to each work. @jcoyne?

**Comment by jcoyne** [comment]
I think this is separate, but will depend upon #1398.  1398 has to do with redirecting to upload_set#edit, whereas this ticket is to add the created works to the upload_set prior to redirecting.
